data/moods.js

```diff
@@ -0,0 +1,7 @@
+export const moods = [
+    { value: 'excited', label: 'Excited'},
+    { value: 'happy', label: 'Happy'},
+    { value: 'meh', label: 'Meh'},
+    { value: 'sad', label: 'Sad'},
+    { value: 'Mad', label: 'Mad'},
```

Comment:
@jameka23 maybe change value to lowercased for consistency?

Reply:
@drteresavasquez ahh good check